canonical-web-and-design/www.ubuntu.com
# Ua4i pricing

## Done
Build the Ua4i pricing page

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /pricing
- Check it matches the [copydoc](https://docs.google.com/document/d/1iOM6oBqsTkptsst3yG-U3sGaYLQqs27go_8yJQdhMBE/edit#)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5009
